### PR TITLE
node and MySql configuration done

### PR DESCRIPTION
**Pull request by Vikas-jangid**
Modules Installed that are required for backend , DataBase Connected.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -8,9 +8,14 @@
...
   },
   "dependencies": {
+    "body-parser": "^1.19.0",
     "core-js": "^3.6.5",
+    "cors": "^2.8.5",
+    "express": "^4.17.1",
     "material-design": "0.0.1",
     "material-design-icons-iconfont": "^6.1.0",
+    "mysql": "^2.18.1",
+    "nodemon": "^2.0.7",
```
Comment (sunil4infobeans) [inline package.json:18]: :+1:

File: server.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+const mysql = require('mysql');
+const app = express();
+const cors = require('cors');
+
+app.use(cors());
+
+
+app.use(bodyParser.json());
+
+
```
Comment (sunil4infobeans) [inline server.js:13]: Pls remove extra new lines

File: server.js [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+var con = mysql.createConnection({
+    host:'localhost',
+    user:'root',
+    password:'Infobeans',
+    database : 'todolist'
```
Comment (sunil4infobeans) [inline server.js:18]: Let's store these credentials in a separate dbCredentials.json file and use it.

File: server.js [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+});
+
+con.connect(function(error){
+    if(error){
+        console.log("Connection Failed");
+        console.log("error is",error);
+    }
+    else{
+        console.log("Connected to Database")
```
Comment (sunil4infobeans) [inline server.js:27]: Use vuetify v-dialog

File: server.js [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+            //  console.log(fields);
+         }
+         else{
+             console.log(err);
+         }
+     })
+
+
+    });
+
```
Comment (sunil4infobeans) [inline server.js:47]: Requires code formatting.

**Review by sunil4infobeans — CHANGES_REQUESTED** [review]
Requires a few updates.

**Comment by sunil4infobeans** [comment]
Good work @Vikas-jangid just add few required updates and it's good to merge.

**Comment by sunil4infobeans** [comment]
@Vikas-jangid remove any mistakenly committed empty file if it's not required.

**Review by Vikas-jangid — COMMENTED** [review]
Separate DB Config file Created And Other Updates are Also Done.

File: server.js (hunk 3 above)
Comment (Vikas-jangid) [reply server.js:18]: Separate DB config is created

File: server.js (hunk 5 above)
Comment (Vikas-jangid) [reply server.js:47]: Code Beautified

File: server.js (hunk 2 above)
Comment (Vikas-jangid) [reply server.js:13]: Extra Line Remove

**Comment by Vikas-jangid** [comment]
Conflicts Resolved Please Review.

**Review by sunil4infobeans — APPROVED** [review]
This looks good to me.

**Pull request merged**